pkg/api/routing/route_register.go

```diff
@@ -156,6 +169,12 @@ func (rr *RouteRegisterImpl) route(pattern, method string, handlers ...web.Handl
 		}
 	}
 
+	if rr.debug && rr.log != nil {
+		// Look twice up the stack as this is a private method called by public methods outside of this package.
+		_, file, line, _ := runtime.Caller(2)
```

Comment:
I thought I had posted this but 🤷‍♀️ 
nit: maybe we only need to log the message only if it's possible to get this information (check the last return value)

Reply:
Great point, I add the check here: [f02c6f3](https://github.com/grafana/grafana/pull/76474/commits/f02c6f3950c793070c9c594a2e07a2c9fbfcec0a)